cartographer_ros_msgs/srv/TrajectoryQuery.srv

```diff
@@ -0,0 +1,20 @@
+# Copyright 2019 The Cartographer Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+int32 trajectory_id
+time min_time
+time max_time
```

Comment:
`min_time` and `max_time` make it more convenient for clients to get only segments they're interested in.
This could be also left out, and let the client filter it.